ci/validate_wheel.sh

```diff
@@ -9,10 +9,10 @@ rapids-logger "validate packages with 'pydistcheck'"
 
 pydistcheck \
     --inspect \
-    "$(echo ${wheel_dir_relative_path}/*.whl)"
+    "$(echo "${wheel_dir_relative_path}"/*.whl)"
```

Comment:
Maybe I'm mistaken, but wouldn't this mean `${wheel_dir_relative_path}` is not within double-quotes since the open double-quote starts before `$(echo ...`?

Reply:
Another `bash` footgun, quoting resets inside of captures